CIPx/CIP-xxxx.md

```diff
@@ -0,0 +1,103 @@
+---
+CIP: ?
```

Comment:
Need a number (12 I believe?)

Reply:
Assigned in https://github.com/cardano-foundation/CIPs/pull/44/commits/250601ead0cac508f89ff12675c2b48dfbe0ef32.